Utilities/OTOLocationHelper.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  LocationHelper.swift
+//  OTOnexus
+//
+//  Created by Nicholas Schlueter on 1/10/18.
+//
+
+import Foundation
+import CoreLocation
+
+class OTOLocationHelper: NSObject {
+    static let shared = OTOLocationHelper()
+    fileprivate let locationManager = CLLocationManager()
+    
+    var currentLocation:CLLocation? {
+        return locationManager.location
+    }
+    
+    var userDeniedLocation:Bool {
+        return CLLocationManager.authorizationStatus() == .denied
+    }
+    
+    override init() {
+        super.init()
+        locationManager.desiredAccuracy = kCLLocationAccuracyBest
+        locationManager.delegate = self
+    }
+    
+    func start() {
+        if CLLocationManager.authorizationStatus() == .notDetermined {
+            locationManager.requestWhenInUseAuthorization()
+        } else if CLLocationManager.authorizationStatus() == .authorizedWhenInUse {
```

Comment:
Are we relying on the consuming app to set the plist values, and authorization scope (always vs when-in-use)? If so, I think a simple `||  CLLocationManager.authorizationStatus() == .always` would do it.

Reply:
They do have to put it in the Plist. I think the only way it would come back as always is if they authorized it themselves first since we are requesting when in use, but I take your point.